LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright The OpenTelemetry Authors
```

Comment:
This line exists in many other files too.

Reply:
Do we need to change them as well?

Reply:
yes, but I can update the PR to fix them as well since I have done the same for other projects already, let me look into that.

Reply:
see https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2429/commits/30e8476b4b1eb560249c7960a13dc13b899d9e96